main.py

```diff
@@ -1,5 +1,7 @@
 #creating a simple function
 def my_function(fname, lname):
-  print(f"{fname} {lname}")
+  print(f"Cheetahs instructor:{fname} {lname}")
+
+
+my_function("Masi", "Atefe")
```

Comment:
print(f"Cheetahs instructor:{fname} {lname}")
```suggestion
my_function("Masi", "Atefe")
```